plugins/com.google.cloud.tools.eclipse.test.logback/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,13 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.test.logback
+Bundle-Version: 0.1.0.qualifier
+Fragment-Host: ch.qos.logback.classic
+Bundle-Vendor: %providerName
+Bundle-Localization: plugin
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ActivationPolicy: lazy
+Import-Package: org.slf4j.impl
+Eclipse-ExtensibleAPI: true
```

Comment:
I looked up for this directive. Question: does this mean the `.test.logback` bundle allow other bundles that have `.test.logback` as a fragment host (i.e., bundles that have `Fragment-Host: ... .test.logback`) to extend APIs being exported by `.test.logback`? Or, does it allow the `.test.logback` bundle to extend APIs exposed by `ch.qos.logback.classic`?